src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -41,6 +43,14 @@ internal static partial class Library
         private static readonly Regex _minutesDetokenizeRegex = new Regex("[\u000A][\u000A]+", RegExFlags);
         private static readonly Regex _secondsDetokenizeRegex = new Regex("[\u0008][\u0008]+", RegExFlags);
         private static readonly Regex _milisecondsDetokenizeRegex = new Regex("[\u000e]+", RegExFlags);
+        private static readonly Regex _tdTagRegex = new Regex("<\\s*([tT][dD])\\s*\\/{0,1}>", RegExFlags);
```

Comment:
>RegExFlags [](http://example.com/codeflow?start=94&length=10)

Instead of having the casing done in the regex string we can use RegexOptions.IgnoreCase in those regexes

Reply:
I am trying to use the same pattern with other Regex where we want to use RegExFlags. should I use IgnoreCase here instead?

Reply:
I believe we can move all of them to use IgnoreCase - it will make it much easier to read

Reply:
updated here.